lib/node_modules/@stdlib/math/base/complex/divide/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env julia
+
+import JSON
+
+# Get the filename:
+file = @__FILE__;
+
+re1 = [2.0 ^ x for x in linspace( -1023, 1023, 9 )];
+im1 = [2.0 ^ x for x in linspace( -1023, 1023, 9 )];
+re2 = [2.0 ^ x for x in linspace( -1023, 1023, 9 )];
+im2 = [2.0 ^ x for x in linspace( -1023, 1023, 9 )];
+
+inre1 = [ a for a in re1, b in im1, c in re2, d in im2 ];
+inim1 = [ b for a in re1, b in im1, c in re2, d in im2 ];
+inre2 = [ c for a in re1, b in im1, c in re2, d in im2 ];
+inim2 = [ d for a in re1, b in im1, c in re2, d in im2 ];
+
+# Unroll the arrays:
+re1 = reshape( inre1, prod( size( inre1 ) ) );
+im1 = reshape( inim1, prod( size( inim1 ) ) );
+re2 = reshape( inre2, prod( size( inre2 ) ) );
+im2 = reshape( inim2, prod( size( inim2 ) ) );
+
+n = size(re1)[1];
+qre = ones(n)
```

Comment:
Haha. Yeah, you wrote it like Python. In general, we try to stay as close to C as we can across languages.